verilog/analysis/verilog_linter.cc

```diff
@@ -141,6 +144,18 @@ void VerilogLinter::Configure(const LinterConfiguration& configuration) {
   for (auto& rule : syntax_rules) {
     syntax_tree_linter_.AddRule(std::move(rule));
   }
+
+  if (!configuration.external_waivers.empty()) {
+    std::string content;
+    if (verible::file::GetContents(configuration.external_waivers, &content)) {
+      lint_waiver_.ApplyExternalWaivers(configuration.ActiveRuleIds(),
+                                        configuration.external_waivers,
+                                        content);
+    } else {
+      LOG(ERROR) << "Unable to read waivers configuration "
```

Comment:
We should probably surface an error like this to the caller, via `absl::Status`.

Reply:
Last request from me.  Either surface an error/status, or call this somewhere else that does.
The expectation in this function is that the `LinterConfiguration` was already validated and well-formed (hence `void` return).